crates/typed-store-macros/src/lib.rs

```diff
@@ -111,7 +111,50 @@ pub fn derive_dbmap_utils(input: TokenStream) -> TokenStream {
         "std::fmt::Debug + serde::Serialize + for<'de> serde::de::Deserialize<'de>";
     let generics_bounds_token: proc_macro2::TokenStream = generics_bounds.parse().unwrap();
 
+    let config_struct_name_str = format!("{}Config", name);
+    let config_struct_name: proc_macro2::TokenStream = config_struct_name_str.parse().unwrap();
+
+    // TODO: use this to disambiguate Store from DBMap when unifying both
+    let _simple_field_type_name: proc_macro2::TokenStream = simple_field_type_name.parse().unwrap();
+
     TokenStream::from(quote! {
+        /// Create config structs for configuring DBMap tables
+        pub struct #config_struct_name {
+            #(
+                pub #field_names : rocksdb::Options,
+            )*
+        }
+
+        impl #config_struct_name {
+            /// Initialize to defaults
+            pub fn init() -> Self {
+                Self {
+                    #(
+                        #field_names : rocksdb::Options::default(),
```

Comment:
Can we call `default_rocksdb_options()` here instead?

Reply:
we absolutely can :)

Reply:
@mwtian does the general idea make sense in the direction we're trying to go?
Or do we want more features?